src/main/java/codeu/model/store/basic/DefaultDataStore.java

```diff
@@ -98,7 +99,8 @@ private void addRandomUsers() {
     Collections.shuffle(randomUsernames);
 
     for (int i = 0; i < DEFAULT_USER_COUNT; i++) {
-      User user = new User(UUID.randomUUID(), randomUsernames.get(i), "password", Instant.now());
+      User user = new User(UUID.randomUUID(), randomUsernames.get(i), BCrypt.hashpw("password", 
```

Comment:
Nit: Line-wrapping is usually done at a higher syntactic level when possible, and the line continuation is indented 4 spaces so that it's clear that it's part of what came before and not a new statement. So the general preference would be for something more like
```
User user = new User(UUID.randomUUID(), randomUsernames.get(i),
    BCrypt.hashpw("password", BCrypt.gensalt()), Instant.now());
```